src/abstractdatatable/abstractdatatable.jl

```diff
@@ -482,19 +511,19 @@ complete_cases!(df::AbstractDataTable)
 
 * `::AbstractDataTable` : the updated version
 
-See also [`complete_cases`](@ref).
+See also [`dropnull`](@ref) and [`completecases`](@ref).
 
 **Examples**
 
 ```julia
 df = DataTable(i = 1:10, x = rand(10), y = rand(["a", "b", "c"], 10))
 df[[1,4,5], :x] = Nullable()
 df[[9,10], :y] = Nullable()
-complete_cases!(df)
+dropnull!(df)
 ```
 
 """
-complete_cases!(df::AbstractDataTable) = deleterows!(df, find(!complete_cases(df)))
+dropnull!(df::AbstractDataTable) = deleterows!(df, find(!completecases(df)))
```

Comment:
It's quite inefficient to have to allocate a vector just for the  `!` operation.You could do `find(!, completecases(df))`, which isn't pretty but saves an allocation.

We could also imagine avoiding any allocation by having a `onlykeeprows!` function which would directly take the logical index, but for now I would say it's not worth it.